app/models/Meeting.php

```diff
@@ -44,6 +44,12 @@ public function __construct($id = null)
             'on_delete' => true,
         );
 
+        $this->has_one['meeting_token'] = array(
```

Comment:
i think this is not necessary anymore
-> TODO: remove all configugration options in constructors